.internal-ci/helm/fog-services-config/values.yaml

```diff
@@ -32,12 +32,6 @@ fogReport:
   signingCert:
     key: ''
     crt: ''
-  configMap:
-    # https://docs.diesel.rs/diesel/r2d2/struct.Builder.html
-    POSTGRES_IDLE_TIMEOUT: '60'
-    POSTGRES_MAX_LIFETIME: '120'
-    POSTGRES_CONNECTION_TIMEOUT: '5'
-    POSTGRES_MAX_CONNECTIONS: '3'
```

Comment:
Are these deletions pertinent?  Or are they just unused?

Reply:
This moved in to the fog-report chart.  Removed this value and configmap so we don't have conflicts when applying the new fog-report chart.